docs/HowTo/Configure/Configure-HA/Sample-Configuration.md

```diff
@@ -0,0 +1,94 @@
+description: Sample Load Balancers 
+<!--- END of page meta data -->
+
+# Sample Load Balancer Configurations 
+
+## AWS 
+
+For AWS, we recommend the Classic Load Balancer. The Classic Load Balancer routes layer 4 and 7. Register
+the Hyperledger Besu instances to the load balancer and use the [liveness endpoint](../../Interact/APIs/Using-JSON-RPC-API.md#readiness-and-liveness-endpoints)
+for health checks. 
+
+Another alterntive is the Application Load Balancer:
```

Comment:
Rename to `Use the Application Load Balancer to get finer grain control.`

Reply:
Done